test/unit/lib/plugins/aws/provider.test.js

```diff
@@ -1584,10 +1584,14 @@ describe('test/unit/lib/plugins/aws/provider.test.js', () => {
     });
 
     it('should fail if `functions[].image` references image from `provider.ecr.images` that has invalid path', async () => {
+      const modulesCacheStub = {
+        'child-process-ext/spawn': sinon.stub().resolves(),
+      };
```

Comment:
There's dedicated [`shouldStubSpawn`](https://github.com/serverless/test/blob/master/docs/run-serverless.md#shouldstubspawn) option, that does just that